apps/financial-aid/web-osk/src/components/FormProvider/FormProvider.tsx

```diff
@@ -29,6 +29,7 @@ export interface Form {
   ledger?: string
   accountNumber?: string
   emailAddress?: string
+  emailEdited: boolean
```

Comment:
Is this needed? Can't we just send the value even though it hasn't been updated?